shipyard_airflow/control/airflow_connections.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import falcon
+import json
+import requests
+
+from .base import BaseResource
+
+# We need to be able to add/delete connections so that we can create/delete
+# connection endpoints that Airflow needs to connect to
+class AirflowAddConnectionResource(BaseResource):
+
+    authorized_roles = ['user']
+
+    def on_get(self, req, resp, action, conn_id, protocol, host, port):
+        # Retrieve URL
+        web_server_url = self.retrieve_config('BASE', 'WEB_SERVER')
+
+        if 'Error' in web_server_url:
+            resp.status = falcon.HTTP_400
```

Comment:
In the case that the config file is missing (which is how this is triggered, I think), the error code should be `500`, because it is a server error, not a client error.